content/en/opentelemetry/compatibility.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Datadog and OpenTelemetry Compatibility
+disable_sidebar: false
+further_reading:
+- link: /opentelemetry/troubleshooting/
+  tag: "Documentation"
+  text: "OpenTelemetry Troubleshooting"
+---
+
+## Overview
+
+Datadog offers multiple setup options to accommodate various use cases, from full OpenTelemetry (OTel) implementations to hybrid setups using both OpenTelemetry and Datadog components. This page covers the compatibility between different setups and supported Datadog products and features, helping you choose the best configuration for your needs.
+
+## Setups
+
+The following setups are supported:
+
+| Setup Type                                         | API         | SDK         | Collector/Agent                         |
+|----------------------------------------------------|-------------|-------------|-----------------------------------------|
+| **Full OpenTelemetry**                             | OTel API    | OTel SDK    | OTel Collector                          |
+| **OTel to Datadog (OTLP)**                         | OTel API    | OTel SDK    | Datadog Agent (OTLP Ingest)             |
+| **OTel API with Datadog SDK and Agent**            | OTel API    | Datadog SDK | Datadog Agent                           |
+| **OTel with embedded Datadog Collector (Preview)** | OTel API    | OTel SDK    | Datadog Agent (with embedded Collector) |
```

Comment:
This is a tricky one. The Agent with embedded OTel collector can accept both OTel SDK -> embedded collector while still accepting DD SDK -> Agent.

(and we position it as Datadog Agent with embedded OTel Collector)

Reply:
Makes sense. I updated the cell to indicate both Datadog and OTel SDK, plus I added a note with more clarification below the table.